### DISCUSSION TITLE
Can the material with multiple principal components obtain the phonon scattering rate through gpumd? If so, which function is it? If not, can the operational details be explained clearly

### DISCUSSION THREAD

**Discussion by 3183427473** [brucefan1983/GPUMD#587] [Q&A] [answered]
Can the material with multiple principal components obtain the phonon scattering rate through gpumd? If so, which function is it? If not, can the operational details be explained clearly

**Comment by brucefan1983** [comment, marked as answer]
For phonon transport in solids, regardless of the chemical composition, one can use the HNEMD method to calculate the diffusive spectral thermal conductivity $\kappa(\omega)$ and use the NEMD method to obtain the ballistic thermal conductance $G(\omega)$. Then one can calcualte the specral phonon mean free path $\lambda(\omega)=\kappa(\omega)/G(\omega)$. If you can calculate the spectral phonon group velocity $v_{\rm g}(\omega)$ by some means, you can get the spectral scattering time $\tau(\omega)=\lambda(\omega)/v_{\rm g}(\omega)$.

As for the interatomic potential that is required to drive the MD simulations, you can always train an NEP model based on quantum-mechanical reference data. 

These are my geneal answers to your questions. If you have more specific questions, I can provide more detailed answers.
